main.js

```diff
@@ -28,29 +31,29 @@ var consolec = require('./core/unsafe/console.js'),
     modesf = require('./core/modes.js'),
     startup = require('./core/startup.js'),
     modes = modesf.listModes(),
-    arguments = process.argv;
+    args = process.argv;
```

Comment:
why the rename? Its just needless noise expecially given its not in a function so doesnt already exist